[2025-10-07T03:20:35.882+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T03:20:35.887+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_price_elt_daily.load_records scheduled__2025-10-06T02:30:00+00:00 [queued]>
[2025-10-07T03:20:35.889+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_price_elt_daily.load_records scheduled__2025-10-06T02:30:00+00:00 [queued]>
[2025-10-07T03:20:35.889+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-07T03:20:35.893+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load_records> on 2025-10-06 02:30:00+00:00
[2025-10-07T03:20:35.896+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=56181) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T03:20:35.897+0000] {standard_task_runner.py:72} INFO - Started process 56184 to run task
[2025-10-07T03:20:35.897+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_price_elt_daily', 'load_records', 'scheduled__2025-10-06T02:30:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmphjysg7p_']
[2025-10-07T03:20:35.898+0000] {standard_task_runner.py:105} INFO - Job 128: Subtask load_records
[2025-10-07T03:20:36.000+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_price_elt_daily.load_records scheduled__2025-10-06T02:30:00+00:00 [running]> on host b99170a1b783
[2025-10-07T03:20:36.047+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_price_elt_daily' AIRFLOW_CTX_TASK_ID='load_records' AIRFLOW_CTX_EXECUTION_DATE='2025-10-06T02:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-06T02:30:00+00:00'
[2025-10-07T03:20:36.047+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-07T03:20:36.051+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-10-07T03:20:36.051+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-07T03:20:36.051+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-10-07T03:20:36.052+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-07T03:20:36.454+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-10-07T03:20:36.455+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-07T03:20:36.455+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-10-07T03:20:36.455+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-07T03:20:37.299+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-07T03:20:37.402+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-07T03:20:37.403+0000] {connection.py:788} INFO - closed
[2025-10-07T03:20:37.484+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-10-07T03:28:22.269+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-07T03:28:22.272+0000] {connection.py:788} INFO - closed
[2025-10-07T03:28:22.340+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-10-07T03:28:22.488+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T03:28:22.504+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-07T03:28:22.505+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_price_elt_daily, task_id=load_records, run_id=scheduled__2025-10-06T02:30:00+00:00, execution_date=20251006T023000, start_date=20251007T032035, end_date=20251007T032822
[2025-10-07T03:28:22.545+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T03:28:22.558+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T03:28:22.559+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-07T03:34:45.884+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-07T03:34:45.893+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_price_elt_daily.load_records scheduled__2025-10-06T02:30:00+00:00 [queued]>
[2025-10-07T03:34:45.897+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_price_elt_daily.load_records scheduled__2025-10-06T02:30:00+00:00 [queued]>
[2025-10-07T03:34:45.897+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-10-07T03:34:45.903+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load_records> on 2025-10-06 02:30:00+00:00
[2025-10-07T03:34:45.907+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=57003) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-07T03:34:45.908+0000] {standard_task_runner.py:72} INFO - Started process 57009 to run task
[2025-10-07T03:34:45.907+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_price_elt_daily', 'load_records', 'scheduled__2025-10-06T02:30:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/stock_price_elt_daily.py', '--cfg-path', '/tmp/tmpjl34j_y7']
[2025-10-07T03:34:45.909+0000] {standard_task_runner.py:105} INFO - Job 138: Subtask load_records
[2025-10-07T03:34:46.027+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_price_elt_daily.load_records scheduled__2025-10-06T02:30:00+00:00 [running]> on host b99170a1b783
[2025-10-07T03:34:46.081+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_price_elt_daily' AIRFLOW_CTX_TASK_ID='load_records' AIRFLOW_CTX_EXECUTION_DATE='2025-10-06T02:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-06T02:30:00+00:00'
[2025-10-07T03:34:46.082+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-07T03:34:46.087+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-10-07T03:34:46.088+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-07T03:34:46.088+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-10-07T03:34:46.089+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-07T03:34:46.529+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-10-07T03:34:46.529+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-07T03:34:46.529+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-10-07T03:34:46.529+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-07T03:34:47.204+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-07T03:34:47.314+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-07T03:34:47.315+0000] {connection.py:788} INFO - closed
[2025-10-07T03:34:47.370+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-10-07T03:42:18.775+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-10-07T03:42:18.778+0000] {connection.py:788} INFO - closed
[2025-10-07T03:42:18.837+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-10-07T03:42:18.958+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-07T03:42:18.978+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-07T03:42:18.979+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_price_elt_daily, task_id=load_records, run_id=scheduled__2025-10-06T02:30:00+00:00, execution_date=20251006T023000, start_date=20251007T033445, end_date=20251007T034218
[2025-10-07T03:42:19.020+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-07T03:42:19.030+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-07T03:42:19.030+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
